### ISSUE TITLE
General: add JITM on the Feedback page

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#7245]
The Contact Form module is one of the Jetpack's auto-activated features. As a result, as soon as you connect the plugin to your WordPress.com account a new menu item appears in your dashboard and on WordPress.com.

That menu doesn't say much though. It's empty, you don't know what it does, and even if you somewhat guess that it's related to the Contact Form, it doesn't explain how to create your own form.

Maybe we should add a notice there, with 2 options:
- Create your own contact form -> linking to a new static page editor and explaining that you have to click on "Add Contact Form"
- Deactivate the Contact Form feature.

cc @jessefriedman 

Related:

**Comment by beaulebens** [comment]
Yep, agree that the blank-canvas experience here is a perfect spot to include some sort of introductory information on how to use it, what it is, and why you're seeing it.
